packages/frontend/lib/index.ts

```diff
@@ -1,26 +1,31 @@
 /*
- * Copyright (c) 2023 Nango, all rights reserved.
+ * Copyright (c) 2024 Nango, all rights reserved.
  */
 import type { PostPublicUnauthenticatedAuthorization } from '@nangohq/types';
+import type { ConnectUIProps } from './connectUI';
+import { ConnectUI } from './connectUI.js';
+import type {
+    ConnectionConfig,
+    ApiKeyCredentials,
+    AppStoreCredentials,
+    AuthErrorType,
+    AuthOptions,
+    AuthResult,
+    BasicApiCredentials,
+    ErrorHandler,
+    OAuth2ClientCredentials,
+    TBACredentials,
+    TableauCredentials,
+    OAuthCredentialsOverride
+} from './types';
+import { AuthorizationStatus, WSMessageType } from './types.js';
+
+export type * from './types';
```

Comment:
do we need this one if we explicitly export all types 3 lines above?

Reply:
unless I'm misunderstanding your point, we are importing types 3 lines above

Reply:
are we not importing all types again then?

Reply:
We export for people using the frontend lib, we import for us. In terms of whether is it efficient, I'm not sure it really matters since imports are reference objects.